plugin.go

```diff
@@ -84,25 +85,27 @@ func (p Plugin) Exec() error {
 }
 
 func message(repo Repo, build Build) string {
-	return fmt.Sprintf("*%s* <%s|%s/%s#%s> (%s) by %s",
+	return fmt.Sprintf("*%s* <%s|%s/%s#%s> (%s) by %s. Changes: %s",
```

Comment:
we should revert this, since it would alter the format of the message of existing drone users which may not be desirable. Instead, you can use the template section to customize and add message